File: code/R/ch07/07.02.R
### 7.2.1 Rの場合

library(tidyverse)
library(caret)

my_data <- cars # データに別名を付ける．

dim(my_data) # データ数の表示
#> [1] 50  2

head(my_data) # 最初の数件の表示
#>   speed dist
#> 1     4    2
#> 2     4   10
#> 3     7    4
#> 4     7   22
#> 5     8   16
#> 6     9   10

psych::describe(my_data) # 基本統計量
#>       vars  n  mean    sd median trimmed   mad min max range  skew kurtosis   se
#> speed    1 50 15.40  5.29     15   15.47  5.93   4  25    21 -0.11    -0.67 0.75
#> dist     2 50 42.98 25.77     36   40.88 23.72   2 120   118  0.76     0.12 3.64

plot(my_data) # 結果は割愛

File: code/py/ch07/07.02.py
### 7.2.2 Pythonの場合

import statsmodels.api as sm
my_data = sm.datasets.get_rdataset('cars', 'datasets').data

my_data.shape # データ数の表示
#> (50, 2)

my_data.head() # 最初の数件の表示
#>    speed  dist
#> 0      4     2
#> 1      4    10
#> 2      7     4
#> 3      7    22
#> 4      8    16

my_data.describe() # 基本統計量
#>            speed        dist
#> count  50.000000   50.000000
#> mean   15.400000   42.980000
#> std     5.287644   25.769377
#> min     4.000000    2.000000
#> 25%    12.000000   26.000000
#> 50%    15.000000   36.000000
#> 75%    19.000000   56.000000
#> max    25.000000  120.000000

my_data.plot('speed', 'dist', kind='scatter') # 結果は割愛

